import "DseSingleSpiritInfo.proto";

message DseUpdateSpiritInfo
{
	repeated DseSingleSpiritInfo		soldiers = 1;
}